pages/css-grid-superposition--qcj5jo/css-grid-superposition--qcj5jo.md

```diff
@@ -0,0 +1,279 @@
+
+```html
+<div class="chart-component">
+  <div class="title">
+    HTTP response codes <button>toggle chart/info</button>
+  </div>
+  <div class="wrapper">
+    <div class="chart">
+      <!-- chart here -->
+    </div>
+    <div class="info">
+      <!-- short text here -->
+    </div>
+  </div>
+</div>
+```
+
+Now that we have this HTML, the CSS solution would look like this:
+
+```css
+.wrapper {
+  /* .wrapper is the closest positioned ancestor of .chart */
+  position: relative;
+}
+
+.chart {
+  /* .chart is removed from normal flow, therefore
+     .wrapper has the exact same size (and position) as .info
+     because .info is the only child left in the normal flow
+     .chart is positioned relatively to .wrapper */
+  position: absolute;
+  /* same position as .wrapper (thus same position as .info) */
+  left: 0;
+  top: 0;
+  /* same size as .wrapper (thus same size as .info) */
+  height: 100%;
+  width: 100%;
+}
+```
+
+If you're still struggling with how the position CSS property works, this technique can be scary but don't worry:
+
+* It's OK. You're not alone!
+* It gets better with time and practice... 
+
+This solution helped us to achieve our goal.
+The size of our component will now always depend on the size of `.info`, even when this panel is hidden and the chart is displayed.
+
+😎 No more layout shift!
+
+## The display grid solution
+
+Now that we have CSS grid support in all evergreen browsers, we can rely on the fact that you can place and superpose multiple elements in the same areas.
+
+> What do you mean by areas?
+
+If you're not familiar with CSS grid, before continuing this article, I would advise you to:
+
+* Watch any talk from [Rachel Andrew](https://twitter.com/rachelandrew). This [recent video](https://www.youtube.com/watch?v=g1osnSY9mSU) she did with [Jason Lengstorf](https://twitter.com/jlengstorf) is really nice.
+* Always have the CSS-Tricks [complete guide to Grid](https://css-tricks.com/snippets/css/complete-guide-grid/) open in a tab.
+* Practice, practice, practice...
+
+Now that you're more familiar with CSS grid, we can get to the "anti layout shift superposition technique".
+Let's take the initial simple template we had earlier:
+
+Let's take a simplified version of the template:
+
+```html
+<div class="chart-component">
+  <div class="title">
+    HTTP response codes <button>toggle chart/info</button>
+  </div>
+  <div class="chart">
+    <!-- chart here -->
+  </div>
+  <div class="info">
+    <!-- short text here -->
+  </div>
+</div>
+```
+
+If we use CSS grid on `.chart-component` like this:
+
+```css
+.chart-component {
+  display: grid;
+  gap: 1rem;
+}
+```
+
+Our component would look like this:
+
+![UI Component displaying some help](./component-grid-all.png)
+
+Thanks to the awesome [CSS Grid Inspector](https://developer.mozilla.org/en-US/docs/Tools/Page_Inspector/How_to/Examine_grid_layouts) from Firefox's DevTools, we have annotations for columns and rows.
```

Comment:
You are using the product name "DevTools", so "'s" is redundant here.

IMO you should either go with 

```suggestion
Thanks to the awesome [CSS Grid Inspector](https://developer.mozilla.org/en-US/docs/Tools/Page_Inspector/How_to/Examine_grid_layouts) from Firefox DevTools, we have annotations for columns and rows.
```

or

```suggestion
Thanks to the awesome [CSS Grid Inspector](https://developer.mozilla.org/en-US/docs/Tools/Page_Inspector/How_to/Examine_grid_layouts) from Firefox's developer tools, we have annotations for columns and rows.
```

Reply:
Their page says DevTools so.